be/src/olap/skiplist.h

```diff
@@ -100,9 +101,11 @@ class SkipList {
 
     // Immutable after construction
     Comparator const compare_;
+    bool _can_dup;
```

Comment:
No. There are three models: `AGG`, `UNIQUE`, `DUP`. 
Here, we mainly distinguish `DUP` from the other two(`AGG`and `UNIQUE`). Because only the `DUP` model, the data rows in the memtable can be duplicated.
 
And on the other hand, `_is_unique` may make other people mistakenly think "is unique model?"

Reply:
_is_unique means the Key in skiplist is whether unique, it's not related Doris data model.

I think skiplist  shouldn't know the Doris data model.

Reply:
I agree the `SkipList` should not know the Doris data model, and that is why I changed the `Insert()`(which is aware of data model previously) and add the `_can_dup` member.

And I use `_can_dup` here, just to emphasize that this value can only be true under a `DUP` model. Other than this, I think `_can_dup` and `_is_unique` are the same, I don’t have a strong opinion.

Reply:
OK. let's keep `_can_dup `.